style.css

```diff
@@ -0,0 +1,1567 @@
+a,
+abbr,
+acronym,
+address,
+applet,
+article,
+aside,
+audio,
+b,
+big,
+blockquote,
+body,
+canvas,
+caption,
+cite,
+code,
+dd,
+del,
+details,
+dfn,
+div,
+dl,
+dt,
+em,
+embed,
+fieldset,
+figcaption,
+figure,
+footer,
+form,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+header,
+hgroup,
+html,
+i,
+iframe,
+img,
+ins,
+kbd,
+label,
+legend,
+li,
+mark,
+menu,
+nav,
+object,
+ol,
+output,
+p,
+pre,
+q,
+ruby,
+s,
+samp,
+section,
+small,
+span,
+strike,
+strong,
+sub,
+summary,
+sup,
+table,
+tbody,
+td,
+tfoot,
+th,
+thead,
+time,
+tr,
+tt,
+u,
+ul,
+var,
+video {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-size: 10px;
+}
+
+article,
+aside,
+details,
+figcaption,
+figure,
+footer,
+header,
+hgroup,
+menu,
+nav,
+section {
+  display: block;
+  clear: both;
+}
+
+::-webkit-scrollbar {
+  display: none;
+}
+
+body {
+  line-height: 1;
+  color: #555;
+}
+
+ol,
+ul {
+  list-style: none;
+}
+
+blockquote,
+q {
+  quotes: none;
+}
+
+blockquote ::after,
+blockquote ::before {
+  content: "";
+  content: none;
+}
+
+q ::after,
+q ::before {
+  content: "";
+  content: none;
+}
+
+table {
+  border-collapse: collapse;
+  border-spacing: 0;
+}
+
+@font-face {
+  font-family: Avalon;
+  src: url(lib/fonts/avalon.ttf);
+}
+
+@font-face {
+  font-family: "Avalon Bold";
+  src: url(lib/fonts/avalon_bold.ttf);
+}
+
+@font-face {
+  font-family: "ARS Maquette";
+  src: url(lib/fonts/ars-maquette-regular.ttf);
+}
+
+@font-face {
+  font-family: Shentox;
+  src: url(lib/fonts/shentox-regular.otf);
+}
+
+html {
+  box-sizing: border-box;
+}
+
+h1,
+h2 {
+  font-family: "Avalon Bold", sans-serif;
+  font-weight: 700;
+}
+
+a,
+figcaption,
+h3,
+h4,
+h6,
+li,
+p {
+  font-family: "ARS Maquette", "Helvetica Neue", Arial, sans-serif;
+}
+
+h1 {
+  font-size: 3.2rem;
+  margin: 1rem 0;
+}
+
+h2 {
+  font-size: 2.8rem;
+}
+
+h3 {
+  font-size: 1.6rem;
+  font-weight: 700;
+  line-height: 2rem;
+  margin: 1rem 0;
+}
+
+.section-title {
+  color: #f42;
+  cursor: pointer;
+}
+
+.section-title:hover {
+  color: #000;
+}
+
+h5 {
+  font-family: Shentox, "Helvetica Neue", Arial, sans-serif;
+  font-size: 1.2rem;
+  font-weight: 400;
+  text-transform: uppercase;
+  letter-spacing: 0.1rem;
+  line-height: 2rem;
+}
+
+a {
+  color: inherit;
+  text-decoration: none;
+}
+
+.first-link {
+  opacity: 1 !important;
+}
+
+li {
+  display: inline-block;
+}
+
+.red-link {
+  font-size: 1.6rem;
```

Comment:
Can we delete this class? :thinking: 

It is redundant as it isn't used anywhere in the `index.html` file.